src/test/java/org/elasticsearch/search/functionscore/ExplainableScriptTests.java

```diff
@@ -86,6 +88,7 @@ public void testNativeExplainScript() throws InterruptedException, IOException,
         for (SearchHit hit : hits.getHits()) {
             assertThat(hit.getId(), equalTo(Integer.toString(idCounter)));
             assertThat(hit.explanation().toString(), containsString(Double.toString(idCounter) + " = This script returned " + Double.toString(idCounter)));
+            assertThat(hit.explanation().toString(), containsString("1.0 = tf(freq=1.0), with freq of"));
```

Comment:
To prevent future changes from removing the nesting, we could add for example
`assertThat(hit.explanation().getDetails().length, equalTo(2));`
or something even more explicit.
@brwe WDYT?